content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -61,6 +61,7 @@ The output contains a section similar to this:
 
 {{< note >}}
 If you use a Docker credentials store, you won't see that `auth` entry but a `credsStore` entry with the name of the store as value.
+In that case you can create secret directly. See [create-a-secret-by-providing-credentials-on-the-command-line](#create-a-secret-by-providing-credentials-on-the-command-line)
```

Comment:
Why do you write a link anchor with so many dashes?

Reply:
ahh, missed it. Thanks for pointing out